main.go

```diff
@@ -1,7 +1,99 @@
 package main
 
-import "fmt"
+import (
+	"./conf"
```

Comment:
use absolute package address `${module_name}/conf` for example: `github.com/aolab/letsgo/conf`

Reply:
done